kem/schemes/schemes.go

```diff
@@ -1,9 +1,20 @@
 // Package schemes contains a register of KEM schemes.
+//
+// Schemes Implemented
+//
+// Based on standard elliptic curves:
+//  HpkeDHKemP256HkdfSha256, HpkeDHKemP384HkdfSha384, HpkeDHKemP521HkdfSha512
```

Comment:
This "Hpke" prefix doesn't match the KEMs exported by `circl/hpke`. Either (1) remove the "Hpke" prefix (here and below) or (2) ename the KEMs in `circl/hpke` to match their names here. My preference would be option (2), since the existing names are somewhat ambiguous in this context.

Reply:
```go
	dhkemP256HkdfSha256.kemBase.id = DHKemP256HkdfSha256
	dhkemP256HkdfSha256.kemBase.name = "HpkeDHKemP256HkdfSha256"
```

KEM _names_ match, what do not match are Identifiers.

However, using `hpke.HpkeDHKemP256HkdfSha256` makes linter unhappy, it argues that `hpke.DHKemP256HkdfSha256` is clear enough.

Reply:
Ack, let's keep the linter happy.